### ISSUE TITLE
MD034 - Bare URL used  >  advice gives code that fails npm/docusaurus md to html conversion/build

### ISSUE THREAD

**Issue by TuxVinyards** [DavidAnson/markdownlint#1616]
The advice given in  [doc/md034.md](doc/md034.md) throws build errors with  `npm` when using it:

> To fix this, add angle brackets around the URL or email address:

```markdown
For more info, visit <https://www.example.com/> or email <user@example.com>.
```

I have found that the following pattern in the Docusaurus templates works better instead:

```markdown
For more info, visit [https://www.example.com/](https://www.example.com/) or email [user@example.com](user@example.com).
```

As in [https://www.example.com/](https://www.example.com/) 

Or  [https://github.com/facebook/docusaurus/blob/main/website/docs/deployment.mdx?plain=1](https://github.com/facebook/docusaurus/blob/main/website/docs/deployment.mdx?plain=1) at line 44

Not sure how much this is a `npm` thing or how much it is Docusaurus.   

Obviously it doesn't happen with GitHub readme files but maybe other .md convertors are affected too?

**Comment by DavidAnson** [comment]
These are both valid syntax for Markdown links, so I do not understand what you mean by "throws build errors with npm". Please explain that and give an example of what you think the problem is.

If you want to enforce a particular style for links and images (like above), you can use MD054 to do so: https://github.com/DavidAnson/markdownlint/blob/main/doc/md054.md
